edenai_apis/apis/lovoai/config.py

```
@@ -1,458 +1,458 @@
+    "he-IL_Yael Haddad": "63b408e6241a82001d51c4fa",
+    "fa-IR_Yasmina Hakimi": "63b40894241a82001d51c420",
+    "zh-CN_Yifu Wu": "63b41845241a82001d51e024",
+    "zh-CN_Yihan Chen": "63b41839241a82001d51e006",
+    "zh-CN_Yinuo Zhang": "63b41837241a82001d51e000",
+    "es-PA_Yolanda Ezequiel": "63b40864241a82001d51c372",
+    "ps-AF_Zahra Qurban": "63b409b7241a82001d51c704",
+    "ar-OM_Zahra Sultan": "63b406b5241a82001d51b0b2",
+    "az_AZ_Zeynab Cafarova": "63b406d1241a82001d51b108",
+    "hu-HU_Zoe Nagy": "63b408f0241a82001d51c518",
+    "pl-PL_Zuzanna Kackz": "63b409af241a82001d51c6ec",
+    "kn-IN_Aadesh Madar": "63b4094b241a82001d51c5fc",
```

Ruff (0.8.2)

Dictionary key literal `"kn-IN_Aadesh Madar"` repeated

Remove repeated key literal `"kn-IN_Aadesh Madar"`

(F601)